<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-20</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DAAB File Compressor/Decompressor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="compressPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decompressPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="txtTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="daabTable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="dragndrop">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Drag and Drop the &quot;.txt&quot; or &quot;.daab&quot; file(s) you wish to process!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.txt Files:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="daabLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.daab Files:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtSelLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="daabSelLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>321</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="txtTableClearPush">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="daabTableClearPush">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size of the Selected File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSizeSelectedFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 bytes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSelectedFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
